voxblox/include/voxblox/test/layer_test_utils.h

```diff
@@ -76,6 +76,37 @@ class LayerTest {
   static constexpr double kTolerance = 1e-10;
 };
 
+template <typename VoxelType>
+void LayerTest<VoxelType>::CompareVoxel(const VoxelType& voxel_A,
+                                        const VoxelType& voxel_B) const {
+  LOG(FATAL) << "Not implemented for this voxel type!";
+}
+
+template <>
+void LayerTest<EsdfVoxel>::CompareVoxel(const EsdfVoxel& voxel_A,
+                                        const EsdfVoxel& voxel_B) const {
+  constexpr double kTolerance = 1e-10;
+
+  CHECK_NEAR(voxel_A.distance, voxel_B.distance, kTolerance);
```

Comment:
@helenol Haha I know why there were no EXPECT_NEAR in that function. As soon as I changed it, there were compilation errors in some tests :P I had to adapt some tests to fix it, could you have a look at the following [commit](https://github.com/ethz-asl/voxblox/commit/a7845955782d047bb9bf98caa303793d5fac6103)?